perfdash/google-gcs-downloader.go

```diff
@@ -120,33 +114,57 @@ func (g *GoogleGCSDownloader) getJobData(wg *sync.WaitGroup, result JobToCategor
 		for categoryLabel, categoryMap := range tests.Descriptions {
 			for testLabel, testDescriptions := range categoryMap {
 				for _, testDescription := range testDescriptions {
-					fileStem := fmt.Sprintf("artifacts/%v_%v", testDescription.OutputFilePrefix, testDescription.Name)
-					artifacts, err := g.GoogleGCSBucketUtils.listFilesInBuild(job, buildNumber, fileStem)
+					filePrefix := fmt.Sprintf("%v_%v", testDescription.OutputFilePrefix, testDescription.Name)
+					searchPrefix := fmt.Sprintf("artifacts/%v", filePrefix)
```

Comment:
Nit: Why this change? I think I preferred the previous version.

Reply:
mostly because `filePrefix` is used later on when constructing the `resultCategory`, I wanted to avoid building the prefix twice so that we cna change it in one place (hopefully) if the filename conventions change

Reply:
OK, makes sense.